src/plone/restapi/configure.zcml

```diff
@@ -38,6 +38,15 @@
       for="Products.CMFPlone.interfaces.ITestCasePloneSiteRoot"
       />
 
+  <genericsetup:registerProfile
+      name="testing-workflows"
+      title="plone.restapi testing-workflows"
+      directory="profiles/testing-workflows"
+      description="Adds sample workflows for testing"
+      provides="Products.GenericSetup.interfaces.EXTENSION"
+      for="Products.CMFPlone.interfaces.ITestCasePloneSiteRoot"
```

Comment:
IMHO if this profile is needed for the tests, it should be moved inside the test folder and the zcml should be included by the test setup.
In that way you will not even need to bother with the `getNonInstallableProfiles`

Reply:
I have followed the existing convention of where the test profiles are put, there is already a `testing` profile in the same directory. In principle i agree but i'd rather not change too much for just this hotfix, so if you don't object i'd prefer to leave it where it is.